src/iroh.udl

```diff
@@ -57,6 +57,7 @@ interface AuthorId {
 interface Entry {
   AuthorId author();
   bytes key();
+  Hash hash();
```

Comment:
Needed to add this to satisfy the API loop provided by feeding `doc.get_content`doc.latest()`.

This is mainly aimed at keeping the API functional. open to revisiting later